CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,21 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: Set<Int> =  [1, 2, 3, 4, 5, 6]
+var numberfull: [Int] = Array<Int>(1...45)
+var lotto: Set<Int> = []
+
+    while lotto.count < 6 {
+        lotto.insert(Int.random(in: 1...45))
+    }
+
+func matchingLotto(myLottoNumbers: Set<Int>, lotto: Set<Int>) {
+    let result: Set<Int> = myLottoNumbers.intersection(lotto)
+    if result.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    }
+    return print(result)
+}
```

Comment:
result 배열이 비어있을 때에는 if 문에 print 문도 실행되고 return 문에 print 문도 실행되고 있는 것 같아요.
if 문 안에 return 을 넣어줘서 조건이 해당되면 함수를 종료시키도록 작성해 볼 수 있을 것 같아요 :)

또한 요구사항 출력문에는 결과를 출력해줄 때 괄호를 출력해 주고 있지 않은데요. 어떻게 하면 형식을 맞춰서 괄호 없이 숫자만 출력할 수 있도록 수정할 수 있을까요?

Reply:
```suggestion
func matchingLotto(myLottoNumbers: Set<Int>, lotto: Set<Int>) {
    let result: Set<Int> = myLottoNumbers.intersection(lotto)
    if result.isEmpty {
        print("아쉽지만 겹치는 번호는 없습니다.")
        return
    }
     print(result.map{String($0)}.joined(separator: ","))
}
```
if문안에 return을 넣어줘서 종료되도록 해봤습니다!
더하여, 괄호출력이 안되는 조건을 이제 이해를 해 구글링을 통해 
joined메서드를 활용하였고, joined매서드를 활용하려면 Stirng타입으로 변환해주기 위해 map메서드를 활용하여 괄호출력이 안되도록 수정해봤습니다!